docs/wiki/framework/cargo-framework.md

```diff
@@ -141,3 +142,20 @@ Note first arg can be a in-game object or a classname of an object type.
  * Example:
  * [object, vehicle] call ace_cargo_fnc_unloadItem
 ```
+
+### 4.6 Remove/ Delete cargo from vehicle 
+
```

Comment:
1) Thank You for such a quick reaction in my request and 
2) thank you for the function ace_cargo_fnc_removeCargoItem
and also 
3) it's a great idea to specify that the  "function should also be able to accept a specific object as well as a classname" :) It may helps a lot (esp all the people who are new in scripts/sqf writing )